src/main.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+class Numberdrome {
+    constructor() {
+        this.numbers = [];
+    }
+
+    addNumber(n) {
+        this.numbers.push(n);
+        return this.numbers
+    }
+
+    removeNumber(n) {
+        return this.numbers.filter(num => num !== n)
+    }
+
+    sum() {
+        if (this.numbers.length !== 0) {
+            return this.numbers.reduce((a, b) => {return a + b})
```

Comment:
no need for curly braces
https://github.com/mate-academy/style-guides/blob/master/javascript.md#arrows--implicit-return